tests/conftest.py

```diff
@@ -204,3 +205,35 @@ def tiny_bert_tokenizer(_session_tiny_bert_tokenizer):
 @pytest.fixture
 def tiny_bert_config(_session_tiny_bert_config):
     return copy.deepcopy(_session_tiny_bert_config)
+
+
+# Note: These methods are an alternative to the tiny_bert fixtures above.
+# Fixtures cannot be used natively as paramaterized inputs, which we require when
+# we wish to run a test across multiple models, one of which is a HuggingFace BERT Tiny.
+# As a workaround, we inject objects into the PyTest namespace. Tests should not directly
+# use pytest.{var}, but instead should import and use the helper copy methods configure_{var}
+# so the objects in the PyTest namespace do not change.
+def pytest_configure():
+    transformers = pytest.importorskip('transformers')
+    config = transformers.AutoConfig.from_pretrained('prajjwal1/bert-tiny')
```

Comment:
I changed to `bert-base-uncased` and just passing in the params to make it small. See my updates to the other configs in `dev`